models/intel/person-detection-0100/description/person-detection-0100.md

```diff
@@ -0,0 +1,55 @@
+# person-detection-0100
+
+## Use Case and High-Level Description
+
+This is a person detector that is based on MobileNetV2
+backbone with two SSD heads from 1/16 and 1/8 scale feature maps and clustered
+prior boxes for 256x256 resolution.
+
+## Example
+
+![](./person-detection-0100.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP                              | 82.52% (internal test set)                |
+| Pose coverage                   | Standing upright, parallel to image plane |
+| Support of occluded persons     | YES                                       |
+| Occlusion coverage              | <50%                                      |
+| Min person height               | 100 pixels (on 1080p)                     |
+| GFlops                          | 0.786                                     |
+| MParams                         | 1.817                                     |
+| Source framework                | Pytorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x256x256] - An input image in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order is BGR.
+
+## Outputs
+
+The net outputs blob with shape: [1, 1, N, 7], where N is the number of detected
+bounding boxes. For each detection, the description has the format:
+  [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`]
```

Comment:
```suggestion
bounding boxes. Each detection has a format
  [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`], where:
```

Reply:
Your suggestion is not very aligned with md pages for other detection models.

Reply:
I agree, but this sentence seems confusing to me. You may leave it aligned for now, only add `where`.